core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -257,21 +293,18 @@ public boolean sync() {
 
       // process the sync result for the original path
       try {
-        mRootPath.traverse();
+        path.traverse();
       } catch (InvalidPathException e) {
         throw new RuntimeException(e);
       }
-    } catch (AccessControlException | BlockInfoException | FileAlreadyCompletedException
+    } catch (BlockInfoException | FileAlreadyCompletedException
         | FileDoesNotExistException | InterruptedException | InvalidFileSizeException
-        | InvalidPathException | IOException e) {
+        | IOException e) {
       LogUtils.warnWithException(LOG, "Failed to sync metadata on root path {}",
           toString(), e);
     } finally {
       // regardless of the outcome, remove the UfsStatus for this path from the cache
-      mStatusCache.remove(mRootPath.getUri());
-      // downgrade so that if operations are parallelized, the lock on the root doesn't restrict
-      // concurrent operations
-      mRootPath.downgradeToRead();
```

Comment:
is this no longer necessary because everything is unlocked?

Reply:
yes